docs/source/debuggin.rst

```diff
@@ -0,0 +1,55 @@
+Common Errors and Warnings
+*********************
+
+A number of errors and warnings can be returned when attempting to solve a model. Many errors and warnings are indicative of particular bugs.
```

Comment:
replace: can be returned / can be raised.

Also, the second sentence doesn't say much, could probably be cut.